### ISSUE TITLE
CPMIP metrics for a1su reported as 0

### ISSUE THREAD

**Issue by LuiggiTenorioK** [BSC-ES/autosubmit-api#96]
In GitLab by @ragurida on Nov 7, 2024, 14:20

All the CPMIP metrics of the a1su except for Parallelization are reported as 0.

![image](https://earth.bsc.es/gitlab/es/autosubmit-api/uploads/5e880f428934a3bfbef4717ac2a3d616/image.png)

I can see that the metrics work by checking exp a1oo for example.

Note: 2nd chunk of experiment a1su has failed, maybe that is why the metrics are not available.

**Comment by LuiggiTenorioK** [comment]
Hi @ragurida,

The performance metrics calculation only takes into account COMPLETED jobs from the SIM section that have run times greater than 0 seconds (since they are faulty ones) and are not considered outliers. None of the jobs from this workflow seems to fulfill these conditions.

**Comment by LuiggiTenorioK** [comment]
In GitLab by @ragurida on Nov 7, 2024, 15:56

I'm pretty sure that I had 1 run where chunk 1 completed successfully in more than 17m on 23.10 at 13:11. 
Not really sure why it's not counted.

Experiment path: /gpfs/scratch/ehpc01/bsc245864/a1su

**Comment by LuiggiTenorioK** [comment]
Yes, according to what is on `20241023_124855_run.log` it did run and was completed in a previous run. It seems to possibly be an internal problem in this version of Autosubmit that didn't succeed in writing the job data.

@dbeltrankyl do you think this issue is related to the zombie processes?

**Comment by LuiggiTenorioK** [comment]
In GitLab by @dbeltrankyl on Nov 8, 2024, 09:35

> @dbeltrankyl do you think this issue is related to the zombie processes?

Not for the zombies itself, but the database fixes made in that branch should fix it

**Comment by mcastril** [comment]
@kinow @dbeltrankyl when testing the new Autosubmit update could you check this too? That the performance metrics are well generated

**Comment by kinow** [comment]
> @kinow @dbeltrankyl when testing the new Autosubmit update could you check this too? That the performance metrics are well generated

Sure thing! I will start a new weekly issue for the testing suite, and document that besides the existing test cases for climatedt, I will include a list of issues that need to be checked, like this one :+1:

**Comment by kinow** [comment]
I'm launching the testing suite this week again, still same run 50 from last week, updating some experiments, and @dbeltrankyl reminded me to test the API issues.

For this one, I looked at `t01s` and the performance metrics are being updated (even though the experiment is failing at one job). Will post the link & screenshot in the DestinE internal repository, in the issue for the testing suite.

**Comment by ntorqulu** [comment]
Should we close this issue @LuiggiTenorioK?
I've checked exp `a1su` in both ClimateDT VM and BSC, and on both places the AS version used to run it is `3.11.0-hotfix` and `4.1.10`, both lower than the minimal maintained AS version for the API.

<img width="1079" height="452" alt="Image" src="https://github.com/user-attachments/assets/78086aa6-90b6-43da-a923-b11bd0504683" />

**Comment by LuiggiTenorioK** [comment]
> Should we close this issue [@LuiggiTenorioK](https://github.com/LuiggiTenorioK)?
> I've checked exp `a1su` in both ClimateDT VM and BSC, and on both places the AS version used to run it is `3.11.0-hotfix` and `4.1.10`, both lower than the minimal maintained AS version for the API.

Probably the experiment was archived or deleted. Most likely, the issue doesn't exist anymore because of the multiple enhancements made to that process in AS in the last 2 years.

And even so, this doesn't seem to be an issue with the API.

Closing...
